chai-fuzzy/.editorconfig

```diff
@@ -0,0 +1,6 @@
+root = true
```

Comment:
Do you have a better argument than it not being in other folders? I don't see that as a good reason at all.